lib/AckGroupingTrackerDisabled.cc

```diff
@@ -22,6 +22,7 @@
 #include "HandlerBase.h"
 #include "PulsarApi.pb.h"
 #include <pulsar/MessageId.h>
+#include "LogUtils.h"
```

Comment:
No. It's just because I changed `std::weak_ptr<ClientImpl>` to `ClientImpl&` in this PR and used forward declaration in the header, so the source file doesn't include "LogUtils.h" (it was included explicitly)

Since these unrelated changes increased the difficulty to review, I'm working on removing them now.